src/cljs/main/broadfcui/page/workspace/method_configs/tab.cljs

```diff
@@ -16,6 +16,13 @@
    [broadfcui.utils :as utils]
    ))
 
+(defn- add-redacted-attribute [config methods]
```

Comment:
I almost prefer that this be `let` at the top of the function that uses it, as there's just the one. Thoughts @MattPutnam? Same for `fake-inputs-outputs` above.

Reply:
I don't have a strong opinion either way.